docs/source/app-dev/submission-disclosed-contracts.rst

```diff
@@ -0,0 +1,75 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _submission-disclosed-contracts:
+
+(Experimental) Submitting commands with disclosed contracts
+###########################################################
+
+As described in :ref:`Divulgence: When Non-Stakeholders See Contracts <da-model-divulgence>`,
+one way of delegating read rights of a contract **C** to a non-stakeholder party **P**
+is by divulging **C** to **P**. However, usage of divulged contracts in submitted commands is deprecated and not compatible with ledger pruning.
+Furthermore, this mechanism requires the contract's stakeholder to explicitly divulge
+the contract to all necessary readers, limiting scalability.
+
+Another alternative for achieving read delegation is to make use of the multi-party submission feature, by including the contract's
+stakeholder in the command submission's ``readAs``. However, this option has downsides as well:
+
+- The granularity is too coarse (e.g. by using a reader party, one discloses everything visible to that party rather than individual contracts).
+- The reader party needs to be co-hosted with the actor party on the same participant, limiting the usage in distributed environments.
+
+Read delegation using disclosed contracts
+-----------------------------------------
+
+The recommended mechanism for achieving read delegation is by using disclosed contracts in the submitted command:
+A party **P**, which is not a stakeholder nor a informee of a contract **C** can act as a submitter of a command which needs to use **C**
+by attaching **C**'s payload as a disclosed contract to the command submission.
+By doing so, the participant will use the attached **C** contract payload for resolving contract and key activeness lookups
+during command interpretation.
+
+This means that usage of a disclosed contract effectively bypasses the visibility restriction of the submitting party's over the
+respective contract. However, the authorization restrictions of the Daml model still apply: the submitted command still needs to be well authorized (i.e. the actors
+need to be properly authorized to execute the action - as described in :ref:`Privacy Through Authorization <da-model-privacy-authorization>`).
+
+If we return to the example depicted in :ref:`Divulgence: When Non-Stakeholders See Contracts <da-model-divulgence>`,
+**ShowIou** is an auxiliary contract used by Alice to fetch the `IOU` under the Painter's
+projection in order to divulge it to him so that the Painter can use it when accepting the `CounterOffer`.
+This auxiliary step can be omitted if instead Alice chooses to share the `IOU`
```

Comment:
May be worth mentioning that this sharing is done off-ledger currently.

Reply:
I see this is covered below (link the word `share` to `How does the submitter get the disclosed contract details`) ?

Reply:
Good point. Linked